indi-sv305/sv305_ccd.cpp

```diff
@@ -277,6 +254,65 @@ bool Sv305CCD::Connect()
         pthread_mutex_unlock(&cameraID_mutex);
         return false;
     }
+    if (isDebug())
+    {
+        // Output camera properties to log 
+        IDLog("Camera Property:\n WxH= %ldx%ld, Color:%d, BayerPattern:%d, MaxBitDepth:%d, IsTriggerCam:%d\n",
```

Comment:
Don't use isDebug + IDLog, use directly LOGF_DEBUG.

Reply:
There are two places where `for` blocks surround the output of debug messages.
If the `isDebug()` condition is eliminated, useless `for` blocks will be executed even when not in debug mode.
I would like to keep the `isDebug()` condition to curb the execution of useless `for` blocks.